Ryujinx.Ava/Ui/Windows/DlcManagerWindow.axaml.cs

```diff
@@ -0,0 +1,264 @@
+        {
+            try
+            {
+                return new Nca(VirtualFileSystem.KeySet, ncaStorage);
+            }
+            catch (Exception ex)
+            {
+                ContentDialogHelper.CreateErrorDialog(this,
+                    string.Format(LocaleManager.Instance[
+                        "DialogDlcLoadNcaErrorMessage"], ex.Message, containerPath));
+            }
+
+            return null;
+        }
+
+        private void AddDlc(string path)
+        {
+            if (!File.Exists(path) || Dlcs.FirstOrDefault(x => x.ContainerPath == path) != null)
+            {
+                return;
+            }
+
+            using (FileStream containerFile = File.OpenRead(path))
+            {
+                PartitionFileSystem pfs = new PartitionFileSystem(containerFile.AsStorage());
+                bool containsDlc = false;
+
+                VirtualFileSystem.ImportTickets(pfs);
+
+                foreach (DirectoryEntryEx fileEntry in pfs.EnumerateEntries("/", "*.nca"))
+                {
+                    using var ncaFile = new UniqueRef<IFile>();
+
+                    pfs.OpenFile(ref ncaFile.Ref(), fileEntry.FullPath.ToU8Span(), OpenMode.Read).ThrowIfFailure();
+
+                    Nca nca = TryCreateNca(ncaFile.Get.AsStorage(), path);
+
+                    if (nca == null)
+                    {
+                        continue;
+                    }
+
+                    if (nca.Header.ContentType == NcaContentType.PublicData)
+                    {
+                        if ((nca.Header.TitleId & 0xFFFFFFFFFFFFE000).ToString("x16") != TitleId)
+                        {
+                            break;
+                        }
+
+                        Dlcs.Add(new DlcModel(nca.Header.TitleId.ToString("X16"), path, fileEntry.FullPath, true));
+
+                        containsDlc = true;
+                    }
+                }
+
+                if (!containsDlc)
+                {
+                    ContentDialogHelper.CreateErrorDialog(this, LocaleManager.Instance["DialogDlcNoDlcErrorMessage"]);
+                }
+            }
+        }
+
+        private void RemoveDlcs(bool removeSelectedOnly = false)
+        {
+            if (removeSelectedOnly)
+            {
+                List<DlcModel> enabled = Dlcs.ToList().FindAll(x => x.IsEnabled);
+
+                foreach (DlcModel dlc in enabled)
+                {
+                    Dlcs.Remove(dlc);
+                }
+            }
+            else
+            {
+                Dlcs.Clear();
+            }
+        }
+
+        public void RemoveSelected()
+        {
+            RemoveDlcs(true);
+        }
+
+        public void RemoveAll()
+        {
+            RemoveDlcs();
+        }
+
+        public async void Add()
+        {
+            OpenFileDialog dialog = new OpenFileDialog() {Title = LocaleManager.Instance["SelectDLCDialogTitle"], AllowMultiple = true};
```

Comment:
```suggestion
            OpenFileDialog dialog = new OpenFileDialog() { Title = LocaleManager.Instance["SelectDLCDialogTitle"], AllowMultiple = true };
```
Same for the one below.